### PR TITLE
Fix/convenience fee error

### PR DESCRIPTION
**Pull request by steveamaza**

### CODE REVIEW

File: public/class-paystack-forms-public.php [hunk 1]
```diff
@@ -62,28 +62,82 @@ public function enqueue_scripts()
 define('KKD_PFF_PAYSTACK_FLATLINE_AMOUNT_PLUS_CHARGE', intval((KKD_PFF_PAYSTACK_LOCAL_CAP-KKD_PFF_PAYSTACK_ADDITIONAL_CHARGE)/KKD_PFF_PAYSTACK_PERCENTAGE));
 define('KKD_PFF_PAYSTACK_FLATLINE_AMOUNT', KKD_PFF_PAYSTACK_FLATLINE_AMOUNT_PLUS_CHARGE - KKD_PFF_PAYSTACK_LOCAL_CAP);
 
-function kkd_pff_paystack_add_paystack_charge($amount)
+class PaystackCharge
```
Comment (ibrahimlawal) [inline public/class-paystack-forms-public.php]: Please namespace this class (prefix with `Kkd_Pff_Paystack_`)

Comment (steveamaza) [reply public/class-paystack-forms-public.php]: Yeaaah. Thanks.

File: public/class-paystack-forms-public.php [hunk 2]
```diff
@@ -62,28 +62,82 @@ public function enqueue_scripts()
...
+    public function add_for_ngn($amountinngn)
+    {
+        return $this->add_for_kobo(ceil($amountinngn * 100)) / 100;
+    }
+}
+
+function kkd_pff_paystack_add_paystack_charge($amount)
+{
+    $feeSettings = Kkd_Pff_Paystack_Public::fetchFeeSettings();
+    $pc = new PaystackCharge(
```
Comment (ibrahimlawal) [inline public/class-paystack-forms-public.php]: Brother, you didn't update it here. 

```suggestion
    $pc = new Kkd_Pff_Paystack_PaystackCharge(
```

Comment (steveamaza) [reply public/class-paystack-forms-public.php]: Fixed

**Pull request merged**